source/client/options_impl.cc

```diff
@@ -179,6 +179,18 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
       "failing execution. Defaults to not tolerating error status codes and connection errors.",
       false, "<string, uint64_t>", cmd);
 
+  std::vector<std::string> h1_connection_reuse_strategies = {"HOT", "FAIR"};
+  TCLAP::ValuesConstraint<std::string> h1_connection_reuse_strategies_allowed(
```

Comment:
This is not particular to this PR or Nighthawk, but it seems kind of crazy there is so much boiler plate involved in adding new commands. I would find it ideal if both Envoy and NH were to take their set of options from a proto and then generate the CLI parser. This could be done with proto annotations etc. It's a pretty big project, so definitely not this PR, but it's a wishlist item.

Reply:
I totally agree. Definitely sounds doable if you ask me.

Reply:
Filed https://github.com/envoyproxy/nighthawk/issues/209